### PR TITLE
Modify Benchmark Configuration for End to End support of the TPCC benchmark

### PR DESCRIPTION
**Pull request by VinaiRachakonda**

### CODE REVIEW

File: .github/scripts/performance-benchmarking/get-mysql-dolt-job-json.sh [hunk 1]
```diff
@@ -43,7 +50,7 @@ echo '
         "containers": [
           {
             "name": "performance-benchmarking",
-            "image": "407903926827.dkr.ecr.us-west-2.amazonaws.com/liquidata/performance-benchmarking:latest",
+            "image": "407903926827.dkr.ecr.us-west-2.amazonaws.com/liquidata/performance-benchmarking:vinai",
```
Comment (VinaiRachakonda) [inline .github/scripts/performance-benchmarking/get-mysql-dolt-job-json.sh:53]: will change back to latest

**Review by coffeegoddd — APPROVED** [review]
lgtm, a couple comments

File: go/performance/utils/tpcc_runner/config.go [hunk 2]
```diff
@@ -229,7 +234,7 @@ func (t *TpccTest) TpccPrepare(ctx context.Context, serverConfig *sysbench_runne
 
 // TpccRun prepares the command executable for the Run step.
 func (t *TpccTest) TpccRun(ctx context.Context, serverConfig *sysbench_runner.ServerConfig, scriptDir string) *exec.Cmd {
-	cmd := sysbench_runner.ExecCommand(ctx, scriptDir+"/tpcc.lua", append(t.getArgs(serverConfig), "run")...)
+	cmd := exec.CommandContext(ctx, scriptDir+"/tpcc.lua", append(t.getArgs(serverConfig), "run")...)
```
[the remark this answers is not in the document]
Comment (VinaiRachakonda) [reply go/performance/utils/tpcc_runner/config.go:237]: Would prefer keeping this. Kept having Stdout issues with .Run()

File: go/performance/utils/sysbench_runner/dolt.go [hunk 3]
```diff
@@ -190,7 +192,7 @@ func getDoltServer(ctx context.Context, config *ServerConfig, testRepo string, p
 
 // sysbenchPrepare returns a exec.Cmd for running the sysbench prepare step
 func sysbenchPrepare(ctx context.Context, test *Test, scriptDir string) *exec.Cmd {
-	cmd := ExecCommand(ctx, "sysbench", test.Prepare()...)
+	cmd := exec.CommandContext(ctx, "sysbench", test.Prepare()...)
```
[the remark this answers is not in the document]
Comment (VinaiRachakonda) [reply go/performance/utils/sysbench_runner/dolt.go:195]: Would prefer keeping

**Pull request merged**